doc/source/ray-core/objects/serialization.rst

```diff
@@ -29,6 +29,15 @@ The numpy array is stored as a read-only object, and all Ray workers on the same
 
 .. tip:: You can often avoid serialization issues by using only native types (e.g., numpy arrays or lists/dicts of numpy arrays and other primitive types), or by using Actors hold objects that cannot be serialized.
 
+Fixing "assignment destination is read-only"
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Because Ray puts numpy arrays in the object store, when deserialized as arguments in remote functions they will become read-only. For example, the following code snippet will crash:
```

Comment:
(Not blocking this PR): is it possible to address the issue on API level?

Reply:
We would have to think about how to do it, but possible. Maybe something like `.options(copy_args=True)` or similar.